audit-triplestore-web/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,16 @@
+#
+# The logging properties used
```

Comment:
Yes, I am suggesting we change both modules to use `logback` instead of `log4j`. 
That can also be a separate JIRA ticket, but I think it is important that we use a consistent logging framework throughout our applications.